examples/null.md

```diff
@@ -0,0 +1,45 @@
+# Null primitive type
```

Comment:
Sorry but I seem to miss this earlier. This is not a Null primitive type. Can you change this heading to `Nullable Type Attribute` and also change the file name to `nullable.md`?